scarb/src/bin/scarb/args.rs

```diff
@@ -226,8 +226,12 @@ pub struct BuildArgs {
     pub test: bool,
 
     /// Which features to enable in code.
-    #[arg(short, long)]
-    pub features: Option<String>,
+    #[arg(short, long, value_delimiter = ',')]
```

Comment:
Is it possible to add multiple delimiters? For instance, we want to support separating features with either comma or space.

Reply:
i think it is not possible
function declaration expects char here
https://docs.rs/clap_builder/4.5.2/src/clap_builder/builder/arg.rs.html#1546-1551

Reply:
so we need to decide if `,` or ` ` to use as delimeter

Reply:
Ok so let's leave it as is